cmake-edit-core/src/main/java/ru/urururu/cmakeedit/core/checker/FunctionSimulator.java

```diff
@@ -0,0 +1,59 @@
+package ru.urururu.cmakeedit.core.checker;
+
+import ru.urururu.cmakeedit.core.CommandInvocationNode;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Map;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'java.util.Map'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)